src/assembler.jl

```diff
@@ -97,23 +120,24 @@ end
 
     current_col = 1
     @inbounds for col in dofs
```

Comment:
Because you then need `Ke[row, current_col] -> Ke[row, tmpi[current_col]]` or something like that.